app/controllers/recipes_controller.rb

```diff
@@ -1,5 +1,46 @@
 class RecipesController < ApplicationController
-  def index; end
+  before_action :authenticate_user!
```

Comment:
### Optional
Because we have this in the application controller it is better to remove this from here.